src/Navigation/Home/index.css

```diff
@@ -0,0 +1,190 @@
+.home {
+  padding: 120px 15px 0px 15px;
+  color: #999999;
+  font-family: sans-serif;
+}
+.top-1 {
+  display: flex;
+  flex-direction: row;
+}
+@media only screen and (max-width: 866px) {
+  .top-1 {
+    flex-direction: column;
+    justify-content: center;
+    padding-left: 30px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.top-2 {
+    flex-direction: column;
+    justify-content: center;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-1 {
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-2 {
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.inside-4 {
+    padding-top: 50px;
+    padding-right: 0px;
+  }
+}
+@media only screen and (max-width: 866px) {
+  div.canvas {
+    padding-top: 50px;
+  }
+}
+@media only screen and (max-width: 550px) {
+  hr {
+    width: 65%;
+    margin-left: 0px;
+  }
+}
+
+@media only screen and (max-width: 550px) {
+  div.canvas {
+    width: 400px;
+  }
+}
+@media only screen and (max-width: 550px) {
+  div.inside-4 {
```

Comment:
should follow BEM naming for css classes, please read following:
- https://css-tricks.com/bem-101/
- https://seesparkbox.com/foundry/bem_by_example
- https://codeburst.io/understanding-css-bem-naming-convention-a8cca116d252